tests/vmi_multus_test.go

```diff
@@ -152,8 +154,17 @@ var _ = Describe("Multus", func() {
 	})
 
 	Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance using different types of interfaces.", func() {
-		Context("VirtualMachineInstance with cni ptp plugin interface", func() {
+		const (
+			CIDR1 = "10.1.1.1/24"
+			CIDR2 = "10.1.1.2/24"
+		)
+		var testIP2 string
+
+		BeforeEach(func() {
+			testIP2 = cidrToIP(CIDR2)
```

Comment:
Why should it be done before each test? I see only one test that is using it.

Reply:
That's a mistake. should have been used more. here for instance. https://github.com/kubevirt/kubevirt/blob/e396329a1d9d6ac929f915b266bce76d5e6b34ce/tests/vmi_multus_test.go#L374 ,but since it is not used much I will use the `CIDR2` constant and the `cidrToIP()` function when necessary.
Is that OK?

Reply:
Sounds reasonable